workflow/rules/common.smk

```diff
@@ -127,20 +163,35 @@ def process_config(config):
         # search for file location
         if config["general"]["virus_base_config"]:
             # shorthand - e.g.: hiv
-            vf = "{VPIPE_BASEDIR}/../config/{VIRUS}.yaml".format(
-                VPIPE_BASEDIR=VPIPE_BASEDIR,
-                VIRUS=config["general"]["virus_base_config"],
-            )
-            if not os.path.exists(vf):
-                # normal search (with normal macro expansion, like the default values)
-                vf = config["general"]["virus_base_config"].format(
-                    VPIPE_BASEDIR=VPIPE_BASEDIR
+            try:
+                vf = cacheopen(
+                    srcdir(
+                        "../../config/{VIRUS}.yaml".format(
+                            VIRUS=config["general"]["virus_base_config"],
+                        ),
+                    ),
+                    "rt",
                 )
-                if not os.path.exists(vf):
-                    raise ValueError(
-                        "Cannot find virus base config",
-                        config["general"]["virus_base_config"],
+            except FileNotFoundError:
+                vf = None
+
+            # normal search (with normal macro expansion, like the default values)
+            if not vf:
```

Comment:
`if vf is not None` is preferred but it will probably not make a difference here.

Reply:
Damn, I keep getting dropped messages today...

So I was saying:
this is my standard way of checking for missing value (so also emptry strings, etc.) except that in this case, the present code _should_ not generate anything beside a None.
(Unlike the reference_file below)

Reply:
(also note that i was `if `*`not`*` vf:`, so the equivalent would be `if vf is None:`)

(to me the first sound like "_if no valid vf _" but it's probably my C and Perl background speaking. Let's make it pythonic anyway).